1217-my_profile_page/352-user_opens_the_page_my_profile.feature

```diff
@@ -3,13 +3,13 @@ Feature: User opens the page My Profile
     @352
     Scenario: User opens the page My Profile
         Given that the user is logged in
-        When they click the menu item My Profile
+        When they click the menu item My user profile
         Then they see My Profile
         And they see their Profile page which includes information for
             | Real name          |
-            | Feide ID           |
+            | ID                 |

```

Comment:
Jeg ville kanskje bevart `Feide ID` her, for det er jo det det er. ID er litt mystisk å kan være ID for hva som helst. Jeg tolker ikke disse navna som ting som nødvendigvis skal være string match til vår app, men heller et mest forklarende navn på felt